go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/Masterminds/semver v1.5.0
 	github.com/Masterminds/semver/v3 v3.2.1
 	github.com/davecgh/go-spew v1.1.1
+	github.com/elliotchance/orderedmap/v2 v2.2.0
```

Comment:
nit: This library does not seem to have much activity. Are we comfortable depending on it?

Reply:
The one we used before didn't have any tests at all...

It's just a little utility. I don't really expect frequent updates if it works.

Reply:
But if you've got a better option, let me know!

Reply:
Actually, I was wrong. It did have tests, they just weren't vendored, which makes sense. 

That said, it's updated even less frequently.

Reply:
Other options include:
- https://github.com/goombaio/orderedmap
- https://github.com/samdolan/go-ordered-map

Both older and not using generics